ast/src/main/scala/jawn/ast/JValue.scala

```diff
@@ -295,6 +303,12 @@ object JArray { self =>
 case class JObject(vs: mutable.Map[String, JValue]) extends JValue {
   final def valueType: String = "object"
 
+  final override def getIterable: Option[Iterable[JValue]] = Some(vs.values)
+  final override def getMap: Option[Map[String, JValue]] = Some(vs.toMap)
```

Comment:
Similarly for `Map`, if you're using Jawn AST, I think the use case would be "go fast or go home", so why not return `mutable.Map` instead of calling `toMap`?